src/ts/components/subtitleoverlay.ts

```diff
@@ -416,3 +418,82 @@ class ActiveSubtitleManager {
     this.activeSubtitleCueCount = 0;
   }
 }
+
+class SubtitleRegionContainerManager {
+  private subtitleRegionContainers: { [region: string]: SubtitleRegionContainer } = {};
+  private subtitleOverlay: SubtitleOverlay;
+
+  /**
+   * @param subtitleOverlay Reference to the subtitle overlay for adding and removing the containers.
+   */
+  constructor(subtitleOverlay: SubtitleOverlay) {
+    this.subtitleOverlay = subtitleOverlay;
+  }
+
+  /**
+   * Removes a subtitle label from a container.
+   * @param labelToRemove
+   */
+  public removeLabel(labelToRemove: SubtitleLabel): void {
+    for (let region in this.subtitleRegionContainers) {
```

Comment:
`const region`

Reply:
or `const region of this.subtitleRegionContainers` and `region.removeLabel(labelToRemove)`

Reply:
`this.subtitleRegionContainers` is currently an object so I cannot use `for... of. I could change it to be a map if you want.

Reply:
Right - my bad. Then it's fine of course :)